packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -149,9 +151,52 @@ export function setValueForProperty(
   if (shouldIgnoreAttribute(name, propertyInfo, isCustomComponentTag)) {
     return;
   }
+
+  if (
+    enableCustomElementPropertySupport &&
+    isCustomComponentTag &&
+    name[0] === 'o' &&
+    name[1] === 'n'
+  ) {
+    let eventName = name.replace(/Capture$/, '');
+    const useCapture = name !== eventName;
```

Comment:
I can hardcode this to `true` and tests still pass. We should have test coverage for `Capture` events firing in the capture phase. We also need test coverage for both bubble + capture listeners on the same element, to verify that they don't clash due to some mistake.

Reply:
Thanks for spotting this! I wrote a test but it was broken :(
I fixed it and now it will fail when useCapture is hard coded to true or false.